.prettierrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
```

Comment:
Prettier is already added as plugin with eslint. So, prettier setup won't be necessary.